docs/references/ic-interface-spec.md

```diff
@@ -1390,6 +1394,18 @@ While an implementation will likely try to keep the interval between the value o
 
 :::
 
+#### On Low Wasm Memory {#on-low-wasm-memory}
+
+A canister can export a function with the name `canister_on_low_wasm_memory`, which is scheduled whenever the canister's remaining wasm memory size in bytes falls from at least a threshold `t` to strictly less than `t`.
+The threshold `t` can be defined in the [canister's settings](#ic-canister_status) and by default it is set to 0.
+
+:::note
+
+While the above function is scheduled immediately once the condition above is triggered, it may not necessarily be executed immediately if the canister does not have enough cycles.
+If the canister gets frozen immediately after the function is scheduled for execution, the function will run once the canister's unfrozen _if_ the canister's wasm memory remains above the threshold `t`.
```

Comment:
```suggestion
If the canister gets frozen immediately after the function is scheduled for execution, the function will run once the canister's unfrozen _if_ the canister's wasm memory remains below the threshold `t`.
```
Shouldn't it mean "below threshold" meaning that the alert only trigger on unfreeze if it is still short of memory?

Reply:
It should indeed be rephrased, but I'd suggest: _if_ the canister's remaining wasm memory size in bytes remains below the threshold `t`.